src/browser/components/MattermostView.jsx

```diff
@@ -223,14 +232,18 @@ const MattermostView = createReactClass({
     if (!this.props.active) {
       classNames.push('mattermostView-hidden');
     }
+
+    const deeplinkingUrl = remote.getCurrentWindow().deeplinkingUrl;
```

Comment:
You should handle which tab should load the deeplink. Otherwise, non-appropriate tabs may also open the link.